wolox-training/igc-rails
# Pundit

### Summary
- Added Pundit dependencies
- Migrated existing access control to Pundit policies
- Fixed existing specs to work with article authors and Pundit.

### Trello card
https://trello.com/c/SkA0Eckw/16-instalar-y-configurar-pundit

### Test plan
1. Create articles, you should be able to edit them
2. Login as another user
3. Try to destroy or edit articles from the first user.
4. It should redirect to the articles listing, showing a notice about not being authorized.
